source/agora/common/crypto/ECC.d

```diff
@@ -146,11 +146,11 @@ public struct Scalar
 
     ***************************************************************************/
 
-    public Scalar invert () @trusted
+    public Scalar invert () const @nogc @trusted
     {
         Scalar scalar = this;  // copy
-        assert(crypto_core_ed25519_scalar_invert(scalar.data[].ptr,
```

Comment:
Whoa. No invert in release mode huh?

This makes me think that we should additionally run the unittests in `-release` mode too (can we?).

Reply:
We could, but realistically, I don't think we are going to use `-release` any time soon.